Tests/UITests/TestApp/SpeechTestView.swift

```diff
@@ -0,0 +1,121 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziSpeechRecognizer
+import SpeziSpeechSynthesizer
+import SwiftUI
+
+
+struct SpeechTestView: View {
```

Comment:
Nice example and great to have here to ensure that this always compiles and allows us to test the functionality every time we make a change 🚀